indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -93,6 +96,14 @@ class BookingDetails extends React.Component {
     this.setState({occurrencesVisible: false});
   };
 
+  showPreBookingWarning = () => {
```

Comment:
It's only used in one place so when merging the state fields you can probably just create a function that sets everything or move the logic directly to the place where you currently call this.